### ISSUE TITLE
Can't encrypt

### ISSUE THREAD

**Issue by heitorsilva** [maebert/jrnl#392]
I'm not being able to encrypt my file, after editing the txt on Sublime Text 3, on Windows 10.

Getting the following error:

```
Traceback (most recent call last):
  File "c:\python27\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27\Scripts\jrnl.exe\__main__.py", line 9, in <module>
  File "c:\python27\lib\site-packages\jrnl\cli.py", line 257, in run
    encrypt(journal, filename=args.encrypt)
  File "c:\python27\lib\site-packages\jrnl\cli.py", line 76, in encrypt
    password = util.getpass("Enter new password: ")
  File "c:\python27\lib\site-packages\jrnl\util.py", line 31, in getpass
    return gp.getpass(prompt)
  File "c:\python27\lib\getpass.py", line 95, in win_getpass
    msvcrt.putch(c)
TypeError: must be char, not unicode
```

**Comment by maebert** [comment]
Fixed for 2.0 with 06e3f0c101cc402139a7741773ce2e881bf7d373

**Comment by makais** [comment]
Is it possible for this fix to be back-ported to version 1, or for a version 2 build to be made available?  Not being able to encrypt on Windows is a showstopper.
